bin/minifollowups/pycbc_injection_minifollowup

```diff
@@ -14,20 +14,96 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
-""" Followup foreground events """
 
-import os, argparse, numpy, logging, h5py, copy
+"""Create a workflow for following up missed loud injections."""
+
+import os
+import argparse
+import logging
+import copy
+import numpy
+import h5py
 import pycbc.workflow as wf
+import pycbc.workflow.minifollowups as mini
+import pycbc.version
+import pycbc.pnutils
 from pycbc.types import MultiDetOptionAction
 from pycbc.events import select_segments_by_definer, coinc
 from pycbc.results import layout
 from pycbc.detector import Detector
-import pycbc.workflow.minifollowups as mini
 from pycbc.workflow.core import resolve_url_to_file
-import pycbc.version, pycbc.pnutils
 from pycbc.io.hdf import SingleDetTriggers, HFile
+from pycbc.conversions import mchirp_from_mass1_mass2
+
 
-parser = argparse.ArgumentParser(description=__doc__[1:])
+def sort_injections(args, inj_group, missed, sort_by):
```

Comment:
Could this be a function in a module? At least the 'calculate sort variable' part. Then it can be shared, and use a single source of truth

Reply:
Sure, any suggestion? `pycbc/workflow/injection.py`? `pycbc/workflow/minifollowups.py`? `pycbc/inject/inject.py`?

Reply:
It doesn't feel like a workflow thing as I think it could be used in the foundmissed plot as well

Would a new module in inject, e.g. `inject/injection_utils.py` make sense here?

Reply:
Ok, I think I found a plausible way to address this and Tom's other comment.

Reply:
I'd say a new module for just one thing is overkill to some extent .. it may become clearer how to refactor this as and when we do find a second use case for the functions.

Reply:
Yeah this can wait then